mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -872,6 +872,8 @@ func resourceSqlDatabaseInstanceCreate(d *schema.ResourceData, meta interface{})
 		instance.Settings = desiredSettings
 	}
 
+	instance.Settings.DeletionProtectionEnabled = d.Get("deletion_protection").(bool)
```

Comment:
This can panic if `settings` has not been configured inside the Terraform code.

Check a couple of lines above, where `instance.Settings` is initialized only under certain condition.
If you reach your added new line and `instance.Settings` hasn't been initialized, the provider will panic.

Reply:
Made changes to fix this.